tests/test_load_results/test_mteb_load_results.py

```diff
@@ -23,3 +46,230 @@ def test_mteb_load_results():
+
+    encoder = MockEncoder()
+    res = example_task.evaluate(encoder, split="test")["default"]
+    res_keys = sorted(list(res.keys()))
+    for task_class in all_subclasses_classes:
+        task_name = task_class.metadata.name
+        result_files = task_files[task_name]
+        for result_file in result_files:
+            with open(result_file, "r") as f:
+                result = json.load(f)
+            assert "scores" in result.keys(), result_file + " not have 'scores'"
+            for subset, subset_scores in result["scores"].items():
+                assert isinstance(subset_scores, list), (
+                    result_file + " 'scores' is not list"
+                )
+                for subset_score in subset_scores:
+                    subset_keys = list(subset_score.keys())
+                    subset_keys.remove("hf_subset")
+                    subset_keys.remove("languages")
+                    subset_keys = sorted(subset_keys)
+                    assert (
+                        res_keys == subset_keys
+                    ), f"{result_file} for task {task_name} keys not matching expected {res_keys}, actual {subset_keys}"
+
+
+def test_load_results_scores_multilabel():
+    task_files = get_all_tasks_results()
+    all_subclasses_classes = all_subclasses(AbsTaskMultilabelClassification)
+    example_task = all_subclasses_classes[0]()
+    example_task.is_multilingual = False
+    example_task.data_loaded = True
+    test_dataset = Dataset.from_dict(
+        {"text": ["test", "test", "test"], "label": [[0, 1], [1, 0], [1, 0]]}
+    )
+    train_dataset = Dataset.from_dict(
+        {"text": ["test"] * 100, "label": [[0, 1], [1, 0]] * 50}
+    )
+    example_task.dataset = DatasetDict(test=test_dataset, train=train_dataset)
+
+    encoder = MockEncoder()
+    res = example_task.evaluate(encoder, split="test")["default"]
+    res_keys = sorted(list(res.keys()))
+    for task_class in all_subclasses_classes:
+        task_name = task_class.metadata.name
+        result_files = task_files[task_name]
+        for result_file in result_files:
+            with open(result_file, "r") as f:
+                result = json.load(f)
+            assert "scores" in result.keys(), result_file + " not have 'scores'"
+            for subset, subset_scores in result["scores"].items():
+                assert isinstance(subset_scores, list), (
+                    result_file + " 'scores' is not list"
+                )
+                for subset_score in subset_scores:
+                    subset_keys = list(subset_score.keys())
+                    subset_keys.remove("hf_subset")
+                    subset_keys.remove("languages")
+                    subset_keys = sorted(subset_keys)
+                    assert (
+                        res_keys == subset_keys
+                    ), f"{result_file} for task {task_name} keys not matching expected {res_keys}, actual {subset_keys}"
+
+
+@pytest.mark.skip
```

Comment:
why skip?

Reply:
In this implementation of test, I have problems with creating evaluation `DenseRetrievalExactSearch.__init__() got multiple values for argument 'model'`. And I left it like that to show my pr at start